client/cypress/integration/dashboard/dashboard_spec.js

```diff
@@ -101,6 +103,34 @@ function dragBy(wrapper, offsetTop = 0, offsetLeft = 0) {
     }));
 }
 
+function resizeBy(wrapper, offsetTop = 0, offsetLeft = 0) {
```

Comment:
I didn't think about it before in the Drag tests, but have you considered having those offsets inverted? My intuition says that the order is "width - height" (Not a good argument, but lmk what you think :laughing:)

Reply:
@gabrieldutra How do you deal with CSS (top-right-bottom-left)? :grin:

Reply:
Yeah I don't know why I did that :) I'll switch em.